### ISSUE TITLE
update Tag color generation to avoid duplicates

### ISSUE THREAD

**Issue by collinbarrett** [collinbarrett/FilterLists#546]
Since https://github.com/collinbarrett/FilterLists/pull/543 , we are starting to see some tags sharing the same color. This was expected at some point, but we'll need to re-think how the tag colors are being generated [here](https://github.com/collinbarrett/FilterLists/blob/master/src/FilterLists.Web/ClientApp/modules/home/components/TagGroup.tsx).

**Comment by Atavic** [comment]
Cannot be added some new color strings? I don't see a pure black. Otherwise, limit/reduce the tags themselves. I don't really like *Clickbait*, it could be fall into *NSFW* tag.

**Comment by Atavic** [comment]
Cannot be added some new color strings? I don't see a pure black. Otherwise, limit/reduce the tags themselves. I don't really like *Clickbait* tag, it could go into *NSFW*.

**Comment by collinbarrett** [comment]
Thanks, @Atavic . Yeah, we can either add more hard-code colors, or use some kind of a function. I got the original list from Kelly's 22 colors from here: https://stackoverflow.com/a/4382138/2343739 We just want new hard-coded colors to be relatively distinct...

I agree that we should try to keep the number of different tags relatively small. However, I think `clickbait` and `nsfw` are pretty different based on how [we've defined them](https://github.com/collinbarrett/FilterLists/blob/master/data/Tag.json).
